tests/tests.jl

```diff
@@ -53,7 +53,36 @@ using ConversionKoans
 end
 
 using ArrayKoans
-@testset "Arrays" begin
+@testset "Array Koans" begin
```

Comment:
A useful criteria for finding test specification boundaries is: 0, 1, 3, n, outside domain.

Reply:
Add some tests trying to find edge conditions and behavior boundaries.